docs/markdown/Commands.md

```diff
@@ -0,0 +1,604 @@
+# Command-line commands
+
+There are two different ways of invoking Meson. First, you can run it directly
+from the source tree with the command `/path/to/source/meson.py`. Meson may
+also be installed in which case the command is simply `meson`. In this manual
+we only use the latter format for simplicity.
+
+Meson is invoked using the following syntax:
+`meson [COMMAND] [COMMAND_OPTIONS]`
+
+This section describes all available commands and some of their Optional arguments.
+The most common workflow is to run [`setup`](#setup), followed by [`compile`](#compile), and then [`install`](#install).
+
+For the full list of all available options for a specific command use the following syntax:
+`meson COMMAND --help`
+
+### configure
+
+```
+$ meson configure [-h] [--prefix PREFIX] [--bindir BINDIR]
+                  [--datadir DATADIR] [--includedir INCLUDEDIR]
+                  [--infodir INFODIR] [--libdir LIBDIR]
+                  [--libexecdir LIBEXECDIR] [--localedir LOCALEDIR]
+                  [--localstatedir LOCALSTATEDIR] [--mandir MANDIR]
+                  [--sbindir SBINDIR] [--sharedstatedir SHAREDSTATEDIR]
+                  [--sysconfdir SYSCONFDIR]
+                  [--auto-features {enabled,disabled,auto}]
+                  [--backend {ninja,vs,vs2010,vs2015,vs2017,vs2019,xcode}]
+                  [--buildtype {plain,debug,debugoptimized,release,minsize,custom}]
+                  [--debug] [--default-library {shared,static,both}]
+                  [--errorlogs] [--install-umask INSTALL_UMASK]
+                  [--layout {mirror,flat}] [--optimization {0,g,1,2,3,s}]
+                  [--stdsplit] [--strip] [--unity {on,off,subprojects}]
+                  [--unity-size UNITY_SIZE] [--warnlevel {0,1,2,3}]
+                  [--werror]
+                  [--wrap-mode {default,nofallback,nodownload,forcefallback}]
+                  [--pkg-config-path PKG_CONFIG_PATH]
+                  [--build.pkg-config-path BUILD.PKG_CONFIG_PATH]
+                  [--cmake-prefix-path CMAKE_PREFIX_PATH]
+                  [--build.cmake-prefix-path BUILD.CMAKE_PREFIX_PATH]
+                  [-D option] [--clearcache]
+                  [builddir]
+```
+
+Reconfigures existing meson project.
```

Comment:
I think this needs more clarification. `reconfigure` has a completely different meaning with other build systems (aka, re-setup) :)

Reply:
Fixed